docs/agent-development.md

```diff
@@ -334,7 +336,25 @@ In addition to each agent having a "native" API for instrumentation, they also i
 
 ### Distributed Tracing
 
-We implement the [W3C Trace Context](https://w3c.github.io/trace-context/) standard, with the exceptions that we currently use the header name "elastic-apm-traceparent" and we do not currently support "tracestate". Both of these exceptions will be removed as Trace Context nears finalisation; we are tracking this in https://github.com/elastic/apm/issues/71.
+We implement the W3C standards, both for HTTP headers and binary fields.
+
+#### Http Headers
+
+Our implementation relies on the [W3C Trace Context](https://www.w3.org/TR/trace-context-1/) standard. Until this standard became final, 
+we used the header name `elastic-apm-traceparent` and we did not support `tracestate`. Soon after the standard became official, we
+started to fully align with it. For backward compatibility reasons, this was done in phases, so that the first step was to look for both
+`traceparent` headers in incoming requests and sending both `traceparent` headers in outgoing requests (with the exception of the RUM agent, 
```

Comment:
Maybe just to be extra explicit we can say
```markdown
so that the first step was to look for both
`traceparent` headers in incoming requests [`elastic-apm-traceparent`, `tracestate`], and sending both these `traceparent` headers in outgoing requests
```

Also, does this mean we send both the `traceparent` headers in outgoing requests, even if only one was received in the incoming request?

Reply:
> Maybe just to be extra explicit

Done.

> Also, does this mean we send both the `traceparent` headers in outgoing requests, even if only one was received in the incoming request?

I think there is a confusion here that proves the need for the more explicit explanation you requested above 🙂 . If you are asking about both `traceparent` and `tracestate` - I assume it depends on whether you are initiating or delegating the `tracestate`. If you are asking about both `elastic-apm-traceparent` and `traceparent`, then yes- if you the downstream agent is old and looks for `elastic-apm-traceparent`, then we need to send it.

Reply:
Yes, it's clear now. Thanks!